index.html

```diff
@@ -137,12 +136,39 @@
     </div>
   </script>
 
+  <!-- Project List template -->
```

Comment:
I'd rename this to "Project List Item" since project list itself sounds more like a list of something but not the individual item. This applies to other "project list" occurrences in this PR as well. (e.g., id, class name, file name, etc)